classes/UpgradeTools/CoreUpgrader/CoreUpgrader80.php

```diff
@@ -65,7 +65,7 @@ protected function upgradeLanguage($lang): void
     {
         $isoCode = $lang['iso_code'];
 
-        if (!\Validate::isLangIsoCode($isoCode)) {
+        if (!\Validate::isLangIsoCode($isoCode) || !\Language::getLangDetails($isoCode)) {
             $this->logger->debug($this->container->getTranslator()->trans('%lang% is not a valid iso code, skipping', ['%lang%' => $isoCode]));
 
             return;
```

Comment:
Coudl be interesting to move this part is the caller, because we won't get any notice in this case when we upgrade to a PrestaShop 1.7